src/main/java/de/teamlapen/vampirism/config/BalanceConfig.java

```diff
@@ -105,6 +107,8 @@
         builder.comment("General options");
         builder.push("general");
 
+        hunterTentDistance = builder.comment("dont set hunterTentDistance <= hunterTentSeperation").defineInRange("hunterTentDistance", 8, 2, Integer.MAX_VALUE);
+        hunterTentSeperation = builder.comment("dont set hunterTentDistance <= hunterTentSeperation").defineInRange("hunterTentSeperation", 3, 1, Integer.MAX_VALUE);
```

Comment:
What happens if one does set the value lower, crash or something else?
If it is something fatal, we could throw an exception somewhere (e.g. `#getBiomeFeatureDistance`) with a meaningful error message

Reply:
Alternatively, we could just fall back to a valid value, if the configuration is invalid